### DISCUSSION TITLE
Is there a way to filter for keys instead of values?

### DISCUSSION THREAD

**Discussion by endolith** [osmandapp/OsmAnd#15762] [Q&A]
I can make a custom POI filter to show only `sport=climbing` and `natural=cliff` and `natural=stone`, but is there a way to show any objects that have a `climbing...=...` tag? Such as `climbing:bolts=2` or whatever?  

I don't think all of them consistently have the `sport=climbing` tag, and they're rare enough that I just want to see all of them on the map.

https://wiki.openstreetmap.org/wiki/Climbing

**Comment by thompehw** [comment]
I run into the limitations of the POI filters quite often, and sometimes use overpass turbo ([example for climbing](https://overpass-turbo.eu/s/1o8P)) in a web browser, export as GPX, open .gpx with OsmAnd (there's a nice import interface), then I can see the results on the map in OsmAnd offline afterwards. It's faster than it sounds if your device can handle quick switching between the apps, and you can save searches in overpass turbo. But it's incomprehensible for general users.

**Reply by endolith** [reply]
Yeah I could do that, but it doesn't stay up to date. The ability to make custom filters is really great, I just want it to be greater. I wonder if there are files I can edit to add these keys as options
